libs/markets/src/lib/components/market-info/info-key-value-table.tsx

```diff
@@ -98,7 +98,7 @@ export const Row = ({
       <div style={{ wordBreak: 'break-word' }}>
         {valueDiffersFromParentMarket ? (
           <div className="flex items-center gap-3">
-            <span className="line-through">
+            <span className="line-through dark:text-vega-dark-300">
```

Comment:
Please, use `text-vega-cdark-300` here.

Reply:
Also it'd be nice to have it light theme friendly, `text-vega-clight-300 dark:text-vega-cdark-300` should do the trick

Reply:
I'll move to using the new c colours as part of a broader ticket